src/components/Common/Header/Header.tsx

```diff
@@ -23,9 +23,7 @@ const Header: React.FC<HeaderProps> = (props: HeaderProps): ReactElement => {
   return (
     <Banner>
       <MobileBanner>
-        <Styledh1>
-          {props.headerTitle ? props.headerTitle : 'QuickPass Web Portal'}
-        </Styledh1>
+        <Styledh1>QuickPass</Styledh1>
```

Comment:
nit: why was this changed?

Reply:
I don't think it makes sense to have it change everytime we go to a new page. Like people will know that you're looking at the dashboard, settings page, etc. Plus it never really updates anyway (not for me that is)